dspace-server-webapp/src/main/java/org/dspace/app/rest/contentreports/ItemFilterUtil.java

```diff
@@ -0,0 +1,283 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.contentreports;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.dspace.content.Bitstream;
+import org.dspace.content.Bundle;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.dspace.services.factory.DSpaceServicesFactory;
+
+/**
+ * Utility methods for applying some of the filters defined in the {@link Filter} enum.
+ *
+ * @author Jean-François Morin (Université Laval) (port to DSpace 7.x)
+ * @author Terry Brady, Georgetown University (original code in DSpace 6.x)
+ */
+public class ItemFilterUtil {
```

Comment:
Again, it'd be nice to have tests for this class.  These might just be basic (Unit) tests, or they could be Integration Tests.  But, obviously tests help with ongoing maintenance so that we can be assured we don't accidentally break anything in future updates.